src/borg/archiver/prune_cmd.py

```diff
@@ -67,6 +74,7 @@ def quarterly_13weekly_period_func(a):
 
 
 def quarterly_3monthly_period_func(a):
+    """Group archives by extracting month number from their creation timestamp"""
```

Comment:
It's not the month number that is ultimately extracted, but instead is the "quarter", measured in groups of three months.